packages/tracing/src/browser/web-vitals/getLCP.ts

```diff
@@ -33,10 +32,11 @@ export interface LargestContentfulPaint extends PerformanceEntry {
   toJSON(): Record<string, string>;
 }
 
-export const getLCP = (onReport: ReportHandler, reportAllChanges = false): void => {
-  const metric = initMetric('LCP');
-  const firstHidden = getFirstHidden();
+const reportedMetricIDs: Set<string> = new Set();
```

Comment:
Keep in mind that `Set` is not available in IE10, although not sure how relevant it is here anyway.